custom_components/powercalc/data/signify/915005988201/model.json

```diff
@@ -0,0 +1,17 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.4:docker"
+    },
+    "name": "Hue Play Gradient Light Tube Large",
+    "standby_power": 0.5,
```

Comment:
Were you able to measure the standby power? This is large for Hue lights. Typically it's between 0.2 and 0.3 W.
When you were not able to measure you can set to 0.25 which would be a good estimate.